src/EditorFeatures/TestUtilities/Threading/WpfTestRunner.cs

```diff
@@ -50,39 +50,67 @@ public sealed class WpfTestRunner : XunitTestRunner
         protected override Task<decimal> InvokeTestMethodAsync(ExceptionAggregator aggregator)
         {
             SharedData.ExecutingTest(TestMethod);
-            var sta = StaTaskScheduler.DefaultSta;
+
+            DispatcherSynchronizationContext synchronizationContext = null;
+            Dispatcher dispatcher = null;
+            var staThreadStartedEvent = new ManualResetEventSlim(initialState: false);
```

Comment:
Do we no longer want to dispose this?

Reply:
Unintentionally included with the revert. Will add back.